src/fcs.jl

```diff
@@ -49,3 +49,42 @@ function pcap_has_fcs(reader::PcapReader)
     end
 end
 
+function _has_fcs(record::PcapRecord)
```

Comment:
This can be exported. We can then make `pcap_has_fcs` a method of `has_fcs` instead, makes more sense. Then implement `has_fcs(reader)` in terms of `has_fcs(record)`. Logically it's a reduction over records with an early exit on first true.
```suggestion
function has_fcs(record::PcapRecord)
```